README.md

```diff
@@ -321,11 +321,39 @@ To install things on the internal MiR PC:
 * connect a monitor and keyboard to the ports that are exposed on one corner of the MiR
 * boot into a live USB linux system
 * `chroot` into the MiR PC
-* download `chrony_2.1.1-1ubuntu0.1_amd64.deb`,
-  `libtomcrypt0_1.17-7ubuntu0.1_amd64.deb`, `libtommath0_0.42.0-1.2_amd64.deb`
-  and `timelimit_1.8-1_amd64.deb` from a PC that has internet and install them
-  in the `chroot` environment onto the MiR PC using `dpkg -i`
+    1. mount MiR partition and bind /dev and /run:
+       ```
```

Comment:
Markdown syntax: `MD031 Fenced code blocks should be surrounded by blank lines`, i.e., insert a blank line above.

There are also several more markdown syntax warnings, which cause the CI test to fail: https://github.com/DFKI-NI/mir_robot/actions/runs/5622161604/job/15234316417?pr=130

See the newly added [README section on pre-commit](https://github.com/DFKI-NI/mir_robot/#pre-commit-formatting-checks) for instructions how to install this locally and fix the errors.

Reply:
Resolved!